cypress/e2e/smoke/dbot/RunMartingaleOld.cy.js

```diff
@@ -1,45 +1,54 @@
 import RunPanel from '../../../support/pageobjects/dbot/run_panel'
 import BotBuilder from '../../../support/pageobjects/dbot/bot_builder_page'
+import BotDashboard from '../../../support/pageobjects/dbot/bot_dashboard_page'
 
-describe('QATEST-99420: Import and run custom strategy', () => {
+describe('QATEST-99420, QATEST-4179: Import and run custom strategy', () => {
+  const size = ['small', 'desktop']
   const botBuilder = new BotBuilder()
   const runPanel = new RunPanel()
+  const dashboard = new BotDashboard()
   let totalPL
 
-  beforeEach(() => {
-    cy.c_login({ user: 'dBot' })
-    cy.c_visitResponsive('/appstore/traders-hub', 'large')
-    cy.c_openDbotThub()
-    cy.c_loadingCheck()
-    cy.c_skipTour()
-    cy.c_switchToDemoBot()
-  })
-
-  it('Run Martingale Old Strategy', () => {
-    botBuilder.openBotBuilderTab()
-    cy.c_skipTour()
-    botBuilder.importStrategyFromToolbar('MartingaleOld')
-    //Enter Expected profit, expected Loss, and Trade Amount
-    cy.window().then((win) => {
-      const martingaleValues = ['5', '4', '1'] //Expected Profit, Expected Loss, Trade Amount
-      let call = 0
-      cy.stub(win, 'prompt').callsFake(() => {
-        return martingaleValues[call++]
+  size.forEach((size) => {
+    it(`Run Martingale Old strategy on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
+      const isMobile = size == 'small' ? true : false
+      const isDekstop = size == 'dekstop' ? true : false
```

Comment:
suggestion: No need for two variables since they are dependent on each other and one will return false if other is true.
```suggestion
      const isMobile = size === 'small'
```